ietf/static/js/timezone.js

```diff
@@ -8,21 +8,17 @@
  names. Time zone can be changed via the select input or by calling the use() method with
  the name of a time zone (or 'local' to guess the user's local timezone).
  */
-window.ietf_timezone; // public interface
-
 (function () {
     'use strict';
     // Callback for timezone change - called after current_timezone is updated
-    var timezone_change_callback;
-    var current_timezone;
+    let timezone_change_callback;
+    let current_timezone;
+    let tz_radios;
+    let tz_selects;
 
     // Select timezone to use. Arg is name of a timezone or 'local' to guess local tz.
     function use_timezone(newtz) {
         // Guess local timezone if necessary
```

Comment:
Seems like this comment should have been deleted along with the following lines that it described.

Reply:
yep, thanks